promscale/page-index/page-index.js

```diff
@@ -91,22 +76,57 @@ module.exports = [
             tags: ["promscale", "install", "analytics"],
             keywords: ["Promscale", "analytics"],
             relatedPages: [
-              "/promscale/:currentVersion:/guides/resource-recomm",
+              "/promscale/:currentVersion:/installation/resource-recomm",
               "/promscale/:currentVersion:/send-data/",
             ],
             excerpt: "Install Promscale on RPM based Linux",
           },
           {
-            title: "tobs",
-            href: "tobs",
-            tags: ["promscale", "install", "analytics", "k8s"],
-            keywords: ["Promscale", "analytics", "Kubernetes"],
+            title: "Docker",
+            href: "docker",
+            tags: ["promscale", "install", "analytics", "docker"],
+            keywords: ["Promscale", "analytics", "Docker"],
             relatedPages: [
-              "/promscale/:currentVersion:/guides/resource-recomm",
+              "/promscale/:currentVersion:/installation/resource-recomm",
               "/promscale/:currentVersion:/send-data/",
             ],
-            excerpt: "Install tobs",
+            excerpt: "Install Docker for Promscale",
           },
+          {
+            title: "Kubernetes",
+            href: "kubernetes",
+            tags: ["promscale", "install", "kubernetes", "analytics", "helm"],
+            keywords: ["Promscale", "Kubernetes", "analytics", "Helm"],
+            relatedPages: [
+              "/promscale/:currentVersion:/installation/resource-recomm",
+              "/promscale/:currentVersion:/send-data/",
+            ],
+            excerpt: "Install Promscale on a Kubernetes cluster",
+            children: [
+            {
+              title: "Promscale on Kubernetes using tobs",
+              href: "tobs",
+              tags: ["promscale", "install", "analytics","tobs"],
+              keywords: ["Promscale", "analytics", "instrumentation"],
+              excerpt: "Install Promscale on Kubernetes with instrumentation ",
+            },
+            {
+              title: "Promscale on Kubernetes using helm charts",
```

Comment:
This page is under `Promscale/Kubernetes/helm`. So I do not think we need to specify the title as `Promscale on Kubernetes using helm`. Mentioning it as `Helm` should be okay!